detect_secrets/core/baseline.py

```diff
@@ -34,13 +34,19 @@ def initialize(
         exclude_lines=exclude_lines_regex,
     )
 
-    if os.path.isfile(path):
+    if not isinstance(path, list):
+        path = [path]
+
+    if check_all_are_files(path):
         # This option allows for much easier adhoc usage.
-        files_to_scan = [path]
-    elif scan_all_files:
-        files_to_scan = _get_files_recursively(path)
+        files_to_scan = path
```

Comment:
If we decide to pursue the option to display errors for non existing files, we should probably change this to a filter.

Reply:
On the new update didn't use the filter as I changed the whole logic! With the new changes, It's still needed? As I'm not checking for errors on the whole list...